<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	color: lightgray;
}

QLabel:disabled{
	color: rgb(100, 100, 100);
}

QWidget{
	background-color: gray;
}

QGroupBox {
	color: lightgray;
	border: 1px solid lightgray;
	border-radius: 5px;
	margin-top: 10px;
}

QGroupBox::indicator:unchecked {
    border: 2px solid lightgray;
	border-radius: 2px;
	background-color: gray;
}

QGroupBox::indicator:checked {
    border: 2px solid rgb(128, 200, 128);
	border-radius: 1px;
	background-color: rgb(68, 140, 68);
}

QGroupBox::indicator:unchecked:disabled {
    border: 2px solid gray;
	border-radius: 4px;
	background-color: rgb(150, 150, 150);
}

QGroupBox::disabled{
	color: rgb(100, 100, 100);
    border: 1px solid rgb(100, 100, 100);
}

QScrollBar::handle {
	border-radius: 5px;
 	border: 2px solid lightgray;
	background-color: gray;
}

QScrollBar::add-page{ 
	border: none;
    background-color: rgb(86, 86, 86);
}

QScrollBar::sub-page {
    background-color: rgb(86, 86, 86);
	border: none;
}

QScrollBar::add-line {
    border: none;
    background-color: rgb(86, 86, 86);
}

QScrollBar::sub-line {
    border: none;
    background-color: rgb(86, 86, 86);
}

QScrollBar:arrow{
	border: none;
	background: none;
	color: none;
}

QListWidget {
    color: lightgray;
    background-color: rgb(86, 86, 86);
	border-radius: 10px;
	padding: 5px;
}

QListWidget:disabled {
    color: rgb(75, 70, 70);
    background-color: rgb(120, 105, 105);
}

QListWidget::item:hover {
	color: rgb(80, 64, 44);
    background: rgb(10, 150, 150);
    border: white;
}

QListWidget::indicator:unchecked {
    border: 2px solid lightgray;
	border-radius: 2px;
	background-color: rgb(86, 86, 86);
}

QListWidget::indicator:checked {
    border: 2px solid rgb(128, 200, 128);
	border-radius: 1px;
	background-color: rgb(68, 140, 68);
}

QListWidget::indicator:unchecked:disabled {
    border: 2px solid rgb(86, 86, 86);
	border-radius: 4px;
	background-color: rgb(110, 110, 110);
}

QListWidget::item:selected {
	color: rgb(80, 25, 44);
	border: none;
	background-color: rgb(79, 255, 52);
}

QListWidget::item:selected:active{
	color: rgb(80, 64, 44);
    background-color: rgb(187, 240, 255);
	border: gray;
	padding-left: 5px;
}

QListWidget::item:selected:!active {
    background: rgb(91, 174, 181);
	border: none;
}

QCheckBox {
	color: white;
    spacing: 5px;
}

QCheckBox::indicator:unchecked {
    border: 2px solid lightgray;
	border-radius: 2px;
	background-color: gray;
}

QCheckBox::indicator:checked {
    border: 2px solid rgb(128, 200, 128);
	border-radius: 1px;
	background-color: rgb(68, 140, 68);
}

QCheckBox:disabled{
    color: rgb(100, 100, 100);
}

QCheckBox::indicator:checked:disabled {
	background-color: rgb(250, 200, 200);
	border-color: rgb(255, 230, 220);
}

QCheckBox::indicator:unchecked:disabled {
    border: 2px solid gray;
	border-radius: 4px;
	background-color: rgb(150, 150, 150);
}

QRadioButton::checked::disabled {
	color: rgb(150, 170, 150);
}

QRadioButton::unchecked::disabled {
	color: rgb(100, 100, 100);
}

QRadioButton::checked {
	color: rgb(167, 255, 238);
}

QRadioButton::unchecked {
	color: lightgray;
}

QRadioButton::indicator::unchecked{ 
	border: 2px solid white;
	border-radius: 6px;
	background-color: white;
	width: 10px;
	height: 10px;
}

QRadioButton::indicator::checked{
    border: 2px solid rgb(128, 200, 128);
	border-radius: 6px;
	background-color: rgb(68, 140, 68);
	width: 12px;
	height: 12px;
}

QRadioButton::indicator::checked::disabled {
	border: 2px solid rgb(150, 150, 150);
	background-color: gray;
}

QRadioButton::indicator::unchecked::disabled {
	border: 2px solid gray;
	background-color: rgb(150, 150, 150);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Episodes</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Scenes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="stepBox">
         <item>
          <property name="text">
           <string>SETUP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ANIM</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkSelectAll">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Select All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="listEpisodes">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="listScenes"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,2">
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Options:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxScript">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Compile Script</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <property name="topMargin">
         <number>25</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Script </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboScript">
          <property name="styleSheet">
           <string notr="true">QComboBox {
	color: black;
    border: 2px solid white;
    border-radius: 6px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: lightgray;
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    width: 20px;
	background-color: rgb(194, 194, 194);
    border-left-width: 2px;
    border-left-color: white;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox:hover{
    border: 2px solid #05B8CC;
    color: gray;
}

QComboBox QAbstractItemView{
    border: 2px solid white;
	border-radius: 10px;
    color: gray;
	background-color: lightgray;
    selection-background-color: #05B8CC;
	selection-color: white;
	padding: 3px 3px ;
}

QComboBox:disabled{
	color: rgb(250, 150, 120);
	background-color: rgb(220, 190, 190);
    border-color: lightgray;
}

QComboBox::drop-down:disabled {
	background-color: gray;
    border-left-color: rgb(150, 150, 150);
}

QComboBox:disabled{
	color: rgb(100, 100, 100);
	background-color: gray;
    border: 2px solid rgb(100, 100, 100);
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxRender">
       <property name="title">
        <string>Render</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_11">
        <property name="topMargin">
         <number>25</number>
        </property>
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioSendFarm">
          <property name="text">
           <string>Send to Farm</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioRenderLocal">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Render Local</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBoxStep">
          <property name="styleSheet">
           <string notr="true">border: null;</string>
          </property>
          <property name="title">
           <string>Render type:</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <property name="topMargin">
            <number>25</number>
           </property>
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioPreComp">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>PreComp</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioComp">
             <property name="text">
              <string>Comp</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxInfo">
       <property name="title">
        <string>Info</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_12" columnstretch="0,0" columnminimumwidth="0,0">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>25</number>
        </property>
        <item row="3" column="1">
         <widget class="QLabel" name="labelInfoRender">
          <property name="text">
           <string>FARM</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Render:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Episode:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelInfoScript">
          <property name="text">
           <string>NONE</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Script:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelInfoSc">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Selected:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelInfoEp">
          <property name="text">
           <string>NONE</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>779</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,1,0">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 200, 114);</string>
       </property>
       <property name="text">
        <string>@leobazao 2022</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="styleSheet">
        <string notr="true">QProgressBar {
	color: lightgray;
	padding: 2 2px;
	border: 2px solid lightgray;
	border-radius: 5px;
}

QProgressBar::chunk {
 background-color: #05B8CC;
 width: 20px;
}</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="format">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushStart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>84</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	color: rgb(72, 72, 72);
    border: 2px solid white;
    border-radius: 6px;
	background-color: lightgray;
    min-width: 80px;
}

QPushButton:pressed {
	color: white;
	border-color: white;
    background-color: #05B8CC;
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QPushButton:hover{
    border: 2px solid #05B8CC;
    color: black;
}

QPushButton::disabled {
	border-color: darkgray;
	color: gray;
}</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose">
       <property name="minimumSize">
        <size>
         <width>84</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	color: rgb(72, 72, 72);
    border: 2px solid white;
    border-radius: 6px;
	background-color: lightgray;
    min-width: 80px;
}

QPushButton:pressed {
	color: red;
    background-color: rgb(255, 196, 197);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QPushButton:hover{
    border: 2px solid #05B8CC;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <pointsize>27</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RENDER Manager</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelVersion">
       <property name="text">
        <string>v.0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>horizontalSpacer</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
